lms/djangoapps/grades/rest_api/serializers.py

```diff
@@ -96,4 +115,4 @@ class SubsectionGradeResponseSerializer(serializers.Serializer):
     course_id = serializers.CharField()
     original_grade = SubsectionGradeSerializer()
     override = SubsectionGradeOverrideSerializer()
-    history = SubsectionGradeOverrideHistorySerializer(many=True)
```

Comment:
If this serializer isn't being used anywhere else, you can go ahead and delete it.